.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: node_js
+node_js:
+  - 'stable'
+  
+install: npm install
+
+script:
+  - npm run lint
+  - npm run build
+  - npm test
```

Comment:
Since we don't have tests lets remove this for now

Reply:
keeping only `npm run lint` and `npm run build`?

Reply:
Yes